FinancialTracker/ViewControllers/TransactionViewController.swift

```diff
@@ -93,3 +119,13 @@ extension TransactionViewController: UIPickerViewDelegate {
         return ""
     }
 }
+
+extension TransactionViewController: RequestOrSendViewControllerDelegate {
+    func showTabBar(sender: RequestOrSendViewController) {
```

Comment:
This is why you should put better names for the functions in delegates, there is a convention, the name of the function should be:

`requestOrSendVIewControllerShowTabBar(_ sender: RequestOrSendViewController)`. 

I just spent five minutes figuring out what this function is. 

Also, I don't see what the point of it is.

Reply:
To disable tab bar while having the popUpView because if i have the popUpView and i click on tabBar and return to Home screen black outs. I will fix name

Reply:
ok